File: sdkwork-app-sdk-go/types/video_detail_vo.go
package types

// 视频详情响应
type VideoDetailVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	VideoId string `json:"videoId"`
	Title string `json:"title"`
	Description string `json:"description"`
	ContentUrl string `json:"contentUrl"`
	Format string `json:"format"`
	Duration int `json:"duration"`
	ThumbnailUrl string `json:"thumbnailUrl"`
	Width int `json:"width"`
	Height int `json:"height"`
	AspectRatio string `json:"aspectRatio"`
	Resolution string `json:"resolution"`
	FileSize int `json:"fileSize"`
	CategoryId int `json:"categoryId"`
	CoverImages ImageMediaResourceList `json:"coverImages"`
	Resource VideoMediaResource `json:"resource"`
	Author AuthorInfo `json:"author"`
	Status string `json:"status"`
	IsPublic bool `json:"isPublic"`
	IsFavorite bool `json:"isFavorite"`
	PlayCount int `json:"playCount"`
	DownloadCount int `json:"downloadCount"`
	LikeCount int `json:"likeCount"`
}

File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/VideoDetailVO.kt
package com.sdkwork.app

data class VideoDetailVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val videoId: String? = null,
    val title: String? = null,
    val description: String? = null,
    val contentUrl: String? = null,
    val format: String? = null,
    val duration: Int? = null,
    val thumbnailUrl: String? = null,
    val width: Int? = null,
    val height: Int? = null,
    val aspectRatio: String? = null,
    val resolution: String? = null,
    val fileSize: Int? = null,
    val categoryId: Int? = null,
    val coverImages: ImageMediaResourceList? = null,
    val resource: VideoMediaResource? = null,
    val author: AuthorInfo? = null,
    val status: String? = null,
    val isPublic: Boolean? = null,
    val isFavorite: Boolean? = null,
    val playCount: Int? = null,
    val downloadCount: Int? = null,
    val likeCount: Int? = null
)
